### PR TITLE
Capture "other" cache related notifications

### PR DESCRIPTION
**Pull request by twe4ked**

### CODE REVIEW

File: lib/peek/views/dalli.rb [hunk 1]
```diff
@@ -49,21 +51,21 @@ def setup_subscribers
...
-        end
-
-        subscribe('cache_write.active_support') do
-          @writes.update { |value| value + 1 }
+          @others.value = 0
         end
 
         subscribe(/cache_(.*).active_support/) do |name, start, finish, id, payload|
+          case name
+          when /^cache_read\./
```
Comment (dewski) [inline lib/peek/views/dalli.rb]: Can we use `\A` here?

**Comment by dewski** [comment]
Is this an existing accessor in the version required by `peek-dalli` or do we need to set a version requirement (there isn't one now)?

File: lib/peek/views/dalli.rb (hunk 1 above)
Comment (sj26) [inline lib/peek/views/dalli.rb]: Actually we could probably use the literal strings?

Comment (twe4ked) [inline lib/peek/views/dalli.rb]: Sounds simpler. :+1:

**Comment by twe4ked** [comment]
I don't think we need a version requirement. If older versions don't post any of the newer (?) events “other” will just show up as 0.

File: lib/peek/views/dalli.rb (hunk 1 above)
Comment (dewski) [inline lib/peek/views/dalli.rb]: Once this is updated I can merge this in.

Comment (twe4ked) [inline lib/peek/views/dalli.rb]: Great, thanks for the quick responses. I've pushed the change.

**Comment by dewski** [comment]
Released in [v1.1.3](https://github.com/peek/peek-dalli/releases/tag/v1.1.3)

**Comment by twe4ked** [comment]
Thanks. :)

**Pull request merged**